src/ess/beer/io.py

```diff
@@ -75,21 +93,76 @@ def _load_beer_mcstas(f, bank=1):
         v = v[0]
         if isinstance(v, bytes):
             v = v.decode()
-        if k in ('mode', 'sample_filename'):
+        if k in ('mode', 'sample_filename', 'lambda'):
             da.coords[k] = sc.scalar(v)
 
+    if 'lambda' in da.coords:
+        da.coords['wavelength_estimate'] = da.coords.pop('lambda')
+
+    if da.coords['wavelength_estimate'].value == '0':
```

Comment:
This is bad because it scatters parameters and associations of modes with parameters over the code. Can you combine these parameters in one place? E.g., a mapping from mode to dict / dataclass of parameters? Then these long if-else chains can be collapsed into dict-lookups.